src/trackedit/internal/au3/au3projecthistory.h

```diff
@@ -19,6 +19,8 @@ class Au3ProjectHistory : public IProjectHistory
 public:
     void init() override;
 
+    muse::async::Notification isUndoRedoAvailableChanged() const override;
```

Comment:
I would suggest to move this method down as it is not the main one for this class